src/app/login/signup/signup.component.ts

```diff
@@ -1,23 +1,25 @@
-import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { Router } from '@angular/router';
 
 import { AuthService } from '@app-services/auth.service';
+import { NotificationsService } from '@app-services/notifications.service';
```

Comment:
Add one file with imports for all services to import from one place
`import { AuthService, NotificationsService } from '@app-services`